pkg/providers/common/common.go

```diff
@@ -26,10 +26,6 @@ const (
 	publicKeyFileName  = "eks-a-id_rsa.pub"
 )
 
-func GetAuditPolicy() string {
```

Comment:
we should remove config/audit-policy.yaml right?

Reply:
yes